json-sql

```diff
@@ -195,7 +236,7 @@ def main():
         print(json.dumps(db.read()))
     elif method == "query":
         queries = json.load(sys.stdin)
-        print(json.dumps([db.execute_query(query) for query in queries]))
+        print(json.dumps([db.execute_query(query) for query in queries], default=datetime_handler))
```

Comment:
Nice usage of default.

If we set default=jsonable, then serializing other types can be done by adding a method to jsonable.